app/Website/database/migrations/2024_03_16_174823_create_blog_posts_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('blog_posts', function (Blueprint $table) {
+            $table->id();
+            $table->string('title');
+            $table->string('slug');
+            $table->longText('excerpt');
+            $table->string('status')->default(\App\Website\Enums\BlogStatusEnum::DRAFT);
+            $table->unsignedBigInteger('author_id');
+            $table->timestamps();
```

Comment:
We would need a publish_date column to control when the article was published / should be published.